### PR TITLE
<DropdownLayout/> - (testkit) fix click outside

### PR DESCRIPTION
**Pull request by EgoziE**

### CODE REVIEW

**Comment by argshook** [comment]
:+1: bubbling didn't cross my mind at all, great work!

File: src/MultiSelect/test/MultiSelect.spec.js [hunk 1]
```diff
@@ -734,5 +734,19 @@ describe('MultiSelect', () => {
         expect(getTagLabelAt(2)).toBe('Alabama');
       });
     });
+
+    describe('click outside', () => {
```
Comment (shlomitc) [inline src/MultiSelect/test/MultiSelect.spec.js:738]: I think it should go to `InputWithOption` since API is coming from there.

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5894/

**Comment by EgoziE** [comment]
It fails interactive visual tests now... :/

**Comment by EgoziE** [comment]
I removed `driver.blur()` from the visual test in order for the dropdown to remain open.

**Pull request merged**